x-pack/plugin/core/template-resources/src/main/resources/ecs@mappings.json

```diff
@@ -180,7 +167,8 @@
             "path_match": [
               "location",
               "*.location"
-            ]
+            ],
+            "match_mapping_type": ["object", "string"]
```

Comment:
Is this needed?

Reply:
Let me maybe remove that, as there are more ways to specify a [`geo_point`](https://www.elastic.co/guide/en/elasticsearch/reference/current/geo-point.html) value, that wouldn't covered by this, for example `"location": [ -71.34, 41.12 ]`.

But as we need to include objects in the `match_mapping_type`, it risks false-positive matches for things like `"foo.location.bar":  "foo"`. But `subobjects: false` does come to the rescue here.